bothan-lib/src/telemetry/metrics.rs

```diff
@@ -0,0 +1,61 @@
+use opentelemetry::{global, KeyValue};
+
+pub struct Metrics{}
+
+impl Metrics {
+    pub fn increment_get_prices_total_requests() {
+        global::meter("server")
+        .u64_counter("get_prices_total_requests")
+        .with_description("total number of requests sent to fetch asset prices")
+        .build().add(1, &[]);
+    }
+
+    pub fn update_get_prices_responses(start_time: i64, success: bool) {
```

Comment:
id much prefer if the function took the actual time delta rather than calculating it itself